test/counts.jl

```diff
@@ -114,3 +114,19 @@ pm = proportionmap(x, weights(w))
 @test pm["a"] ≈ (5.5 / 13.5)
 @test pm["b"] ≈ (4.5 / 13.5)
 @test pm["c"] ≈ (3.5 / 13.5)
+
+# testing small bits type
+bx = [true, false, true, true, false]
+@test countmap(bx) == Dict(true => 3, false => 2)
+
+uint8x = UInt8[0, 10, 255, 34, 10]
```

Comment:
Better write a loop to avoid repeating values.

Reply:
what do you mean? I though the tests as they are are quite straight-forward and easy to understand and change.